networkx/drawing/tests/test_pydot.py

```diff
@@ -28,7 +28,12 @@ def build_graph(self, G):
 
     def assert_equal(self, G1, G2):
         assert_true( sorted(G1.nodes())==sorted(G2.nodes()) )
-        assert_true( sorted(G1.edges())==sorted(G2.edges()) )
+        if G1.is_directed():
+            assert_true(sorted(G1.edges()) == sorted(G2.edges()))
+        else:
+            e1 = sorted(map(sorted, G1.edges()))
```

Comment:
I don't know of this idiom being used often in networkx. This test code feels awkward to me because it is split over many functions. The typical method we've used in other places is to check if the nodes and/or edges are a specified list/set rather than checking if two graphs have the same nodes and edges.  But I would guess that many of the tests do compare the sorted nodes/edges to a specific list.  So sorting seems to be a potential problem. 

Also the nose tools provide ```assert_equal```, so we should pick a better name for this function.
Should we compare sets instead?

    assert_equal(set(G1), set(G2))
    assert_equal(set(map(set, G1.edges())), set(map(set, G2.edges())))